catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -425,7 +436,7 @@ private Boolean containsGzip(Request request) {
     return false;
   }
 
-  private void attachFileToResponse(
+  private Object attachFileToResponse(
```

Comment:
this could be changed to `Map` if you change the return on https://github.com/codice/ddf/pull/6519/files#diff-e0c87c8194aef5fecfb00b3e7c9ae491274cbd8682430ac268e7721242d6ce40R484

unless theres a particular reason you're returning empty string here. in general we should use as specific of a return type as possible.

Reply:
agreed 👌